srcpkgs/jamulus/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'jamulus'
+pkgname=jamulus
+version=3.6.2
+revision=1
+_version=r$(echo ${version} | sed 's/\./_/g')
```

Comment:
```suggestion
_version=r${version//./_}
```

Reply:
This... kind of blew my mind. Thanks for the tip! :)